### ISSUE TITLE
Implement Designs in Front-End

### ISSUE THREAD

**Issue by blackforestboi** [WorldBrain/Research-Engine#25]
Implmentation of the Designs as HTML5/JS interface that runs completely locally. 

Also important to connect the fields with the query: https://github.com/WorldBrain/Research-Engine/issues/26

**Comment by lewizho99** [comment]
yes i agree, with you @oliversauter but who solutions do you propose?

will the html code be generated with js?
